soroban-sdk/src/storage.rs

```diff
@@ -382,6 +382,21 @@ impl Instance {
         )
         .unwrap_infallible();
     }
+
+    pub fn extend_other_contract_ttl(
```

Comment:
This function is not restricted to 'other' contracts, it can as well bump the current contract.

Reply:
Ah yeah that's a good point. I'll move it